### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by IgnacioColuccio**
### Summary
- Added i18n for the blog
- Added a dropdown select to change the current language
- Installed gems to internationalize both Devise and the default rails messages.

### Trello card
https://trello.com/c/Tl8lroCQ/10-internacionalizar-el-blog

### Test plan
1. Use the language dropdown in any page
2. All following pages should fully be in that language (including errors)

### CODE REVIEW

File: app/views/comments/_comment.html.slim [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
   = comment.body
 
 
-= link_to 'Destroy Comment', [comment.article, comment],
+
+= link_to t(:destroy), [comment.article, comment],
                method: :delete,
-               data: { confirm: 'Are you sure?' } 
\ No newline at end of file
+               data: { confirm: t(:are_you_sure) }
```
Comment (Dinuuu) [inline app/views/comments/_comment.html.slim]: Try to always leave a new line at the end of a file

File: app/views/comments/_form.html.slim [hunk 2]
```diff
@@ -8,4 +8,4 @@
     br
     = f.text_area :body
   p
-    = f.submit
\ No newline at end of file
+    = f.submit t(:save)
```
Comment (Dinuuu) [inline app/views/comments/_form.html.slim]: idem

File: config/locales/en.yml [hunk 3]
```diff
@@ -20,4 +20,38 @@
...
+      comment: 
+        one: "Comment"
+        other: "Comments"
+    attributes:
+      article:
+        text: "Text"
+        title: "Title"
+      comment:
+        body: "Body"
+        commenter: "Commenter"
```
Comment (Dinuuu) [inline config/locales/en.yml]: idem

File: config/locales/es.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      comment: 
+        one: "Comentario"
+        other: "Comentarios"
+    attributes:
+      article:
+        text: "Contenido"
+        title: "Título"
+      comment:
+        body: "Contenido"
+        commenter: "Autor"
```
Comment (Dinuuu) [inline config/locales/es.yml]: idem

**Comment by Dinuuu** [comment]
Correct those little comments, squash commits and :+1: . Remember that you would need to rebase your bootstrap branch if it was a branch of this branch

**Comment by IgnacioColuccio** [comment]
Merging without squashing

**Pull request merged**